test/JDBC/input/datatypes/TestSpatialPoint-vu-verify.sql

```diff
@@ -1,29 +1,29 @@
 DECLARE @point geometry;
 SET @point = geometry::STPointFromText('POINT(-122.34900 47.65100)', 4326);
-SELECT STAsText(@point);
+SELECT @point.STAsText();
```

Comment:
Please add the following testcases:
1. combine a geospatial column ref call, geospatial function call, table column, function call
2. use a select using sp_executesql
3. try with []
4. test both single-db and multi-db scenario

Reply:
Done